tests/installation/add_serial_console.pm

```diff
@@ -39,7 +39,7 @@ sub run {
         assert_script_run('sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=-1/\' /etc/default/grub');
     }
     assert_script_run('echo GRUB_TERMINAL_OUTPUT=\"serial gfxterm\" >> /etc/default/grub');
-    assert_script_run('echo GRUB_SERIAL_COMMAND=\"serial\" >> /etc/default/grub');
+    assert_script_run('echo GRUB_SERIAL_COMMAND=\"serial --unit=0\" >> /etc/default/grub');
```

Comment:
Nan, is it fine to set it globally? There is no specific condition for unit=0.

Reply:
BTW, would you please add a comment with bug id to explain where it comes the need?

Reply:
As per the description in the official doc "The serial unit 0 is usually called ‘COM1’, so, if you want to use COM2, you must specify ‘--unit=1’ instead", can we give a parameter in the job, like "GRUB_SERIAL_UNIT=xx"?

https://www.gnu.org/software/grub/manual/grub/grub.html#Serial-terminal
https://www.gnu.org/software/grub/manual/grub/grub.html#serial

Reply:
I checked this is a new parameter 'GRUB_SERIAL_UNIT' introduced in all test modules.

Reply:
Great. Would you please  read https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18930#discussion_r1544153967?

Reply:
OK, updated.